src/clr-angular/layout/tabs/tab.spec.ts

```diff
@@ -15,12 +15,28 @@ import { TABS_ID_PROVIDER } from './tabs-id.provider';
 import { ClrTabsModule } from './tabs.module';
 
 @Component({
+  selector: 'test-wrapper',
   template: `
+        <ng-container #tabContentViewContainer></ng-container>
+   `,
+})
+class TestWrapper {
+  @ViewChild('tabContentViewContainer', { read: ViewContainerRef })
+  set tabContentViewContainer(value: ViewContainerRef) {
+    this.tabsService.tabContentViewContainer = value;
+  }
+  constructor(private tabsService: TabsService) {}
```

Comment:
Changes look good to me, only request would maybe we put a comment in the tab or test describing that this is a fix due to a lifecycle issue. It adds necessary complexity that without context of the original issue may not be clear.

Reply:
Added a few lines in tab-content, where the view is created.